apps/web/src/pages/studies/[studyId]/edit.tsx

```diff
@@ -68,9 +63,7 @@ export default function EditStudy({ study }: EditStudyProps) {
 
           <hr className="govuk-section-break govuk-section-break--l govuk-section-break--visible" />
 
-          <div className="govuk-inset-text">
-            {study.route === 'Commercial' ? COMMERCIAL_GUIDANCE_TEXT : NON_COMMERCIAL_GUIDANCE_TEXT}
-          </div>
+          <div className="govuk-inset-text">{GENERIC_STUDIES_GUIDANCE_TEXT}</div>
```

Comment:
`{GENERIC_STUDIES_GUIDANCE_TEXT}` is just text now that the logic has been removed.
Why not just inline the text in the view?

Reply:
I think this was using the same approach I took of keeping long hardcoded strings in a separate file as I find they make the code quite noisy but not too precious over this!

Reply:
I'm easy with either, but I do like the less busy screen!